### DISCUSSION TITLE
Backup & Restore swarm

### DISCUSSION THREAD

**Discussion by grzesart1978** [BretFisher/ama#163] [Q&A]
Hi Bret,

I would ask about backup swarm  "docker ce" , what is your recomendation. Last week I had problem with cluster and I would like to prevent in future:). Thank you advance.

https://success.docker.com/article/backup-restore-swarm-manager

Best regards
Grzegorz

**Comment by BretFisher** [comment]
That link is what I refer people to. Do you have questions outside what that doc details?
